src/main/java/com/github/vkorobkov/jfixturescmd/CmdArgs.java

```diff
@@ -0,0 +1,94 @@
+package com.github.vkorobkov.jfixturescmd;
+
+import com.github.lalyos.jfiglet.FigletFont;
+import com.github.vkorobkov.jfixtures.JFixtures;
+import com.github.vkorobkov.jfixtures.fluent.JFixturesResultImpl;
+import lombok.SneakyThrows;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.cli.*;
+
+import java.lang.reflect.Method;
+
+import static com.github.vkorobkov.jfixturescmd.Prompt.getProperties;
+
+@Slf4j
+class CmdArgs {
+    private String[] args = null;
+    private Options options = new Options();
+
+    CmdArgs(String[] args) {
+        this.args = args;
```

Comment:
The good practice is to always copy an array since it could be modified from outside